templates/base/footer_content.tmpl

```diff
@@ -15,6 +15,17 @@
 		{{end}}
 	</div>
 	<div class="right-links" role="group" aria-label="{{ctx.Locale.Tr "aria.footer.links"}}">
+		<div class="ui dropdown upward">
+			{{$currentTheme := ThemeName .SignedUser}}
+			<span class="flex-text-inline">{{svg (ThemeIcon $currentTheme) 14}} {{$currentTheme}}</span>
+			<div class="menu theme-menu">
+				{{range Themes}}
```

Comment:
If we do that, I think it might also be a good idea to sort the themes alphabetically beforehand (once, on Gitea startup).
Otherwise, we might confuse users if the order changes because the admin changed the value of the setting.

Reply:
Yes, sorting is a good idea, can add. Not sure if really warranted to store in-memory because sorting a small array is fast, but can do.

Reply:
Right, we can just sort `setting.UI.Themes` after config parsing.